internal/grpc/client.go

```diff
@@ -0,0 +1,41 @@
+package grpc
+
+import (
+	"context"
+	"fmt"
+
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
+)
+
+// Client is a wrapper type around a client connection to the gRPC server
+// used to power outflux.
+type Client struct {
+	cc *grpc.ClientConn
+	OutfluxClient
+}
+
+// NewClient returns a new OutfluxClient connected to the server at the address
+// given as a parameter.
+//
+// TODO(george-e-shaw-iv): Expand this to allow options to be passed and be secure
+// by default.
+func NewClient(ctx context.Context, address string) (*Client, error) {
+	cc, err := grpc.DialContext(ctx, address, grpc.WithTransportCredentials(insecure.NewCredentials()))
```

Comment:
If we're using K8s we can just allow linkerd to do transit security for us. If we're not.... then yes we need to support TLS.

Reply:
I want to make this agnostic to any deployment solution - I'll probably make it configurable. For us, yeah we'll probably leverage linkerd